code/game/objects/structures/displaycase.dm

```diff
@@ -35,10 +35,19 @@
 	QDEL_NULL(showpiece)
 	return ..()
 
+/obj/structure/displaycase/emag_act(mob/user)
+	if(!emagged)
+		to_chat(user, "<span class ='warning'>You override the ID lock on the [src].</span>")
```

Comment:
Remove the space here so that it's consistent with the rest of them. LGTM otherwise.
```suggestion
		to_chat(user, "<span class='warning'>You override the ID lock on the [src].</span>")
```

Reply:
Can remove `the` there. Byond should add that itself if needed

Reply:
That's not on the emag, that's on the display case